### PR TITLE
docs: update CONTRIBUTING and links

### PR DESCRIPTION
**Pull request by benjie**
- Review references to Facebook and adjust as necessary
- Review old links to `facebook.github.io` and update as necessary
- Remove `packages/graphql-language-service/CONTRIBUTING.md` since there was too much overlap with `CONTRIBUTING.md`
- Remove references to a CLA
- Split release instructions into their own file (since most contributors don't need to think about that)
- Formatting changes from running `prettier` against the markdown files (I would like for us to standardise on this, but that's a topic for another PR)

### CODE REVIEW

File: packages/graphiql/README.md [hunk 1]
```diff
@@ -350,4 +356,4 @@ fragment HumanFragment on Human {
 }
 ```
 
-Read more from [GraphQL Fragment Specification](http://facebook.github.io/graphql/#sec-Language.Fragments).
+Read more from [GraphQL Fragment Specification](https://graphql.github.io/graphql-spec/).
```
Comment (benjie) [inline packages/graphiql/README.md]: Oops; I seem to have dropped the fragment - that will need fixing.

**Comment by benjie** [comment]
For sure; I’d also like to add project-specific guidance to CONTRIBUTING (or at least where to go to find it). Are you able to write up the commitlint stuff? You’re much more familiar than I. Separate PR sounds sensible.

**Comment by benjie** [comment]
O_o codecov... these are markdown files

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+You'll need:
+
+- `GH_TOKEN` : a github user token as an environment variable
+- `git` command line installed, obviously!
+- to run `yarn adduser` first to ensure you're authenticated for publishing
+- your npm 2FA should be enabled, and you should have your second factor device handy for publish
+- your remote should be named `origin` for github, and should be the ssh url
+- (coming soon) GPG key uploaded to account for signing
+
+Note: Ideally we can avoid publishing from any branch but `master`, but we can always `--allow-branch mybranch` in case of an emergency for pre-releases. _Whenever you can, always publish from `master`_.
```
Comment (acao) [inline RELEASING.md]: add note about publishing, for example, patch versions off of previous minor/major versions is possible using tags using any of the release?

**Pull request merged**